vendor/symfony/lib/i18n/sfMessageSource_XLIFF.class.php

```diff
@@ -54,6 +54,15 @@ public function &loadData($filename)
     if (!$xml = simplexml_load_file($filename))
     {
       $error = false;
+      $xmlErrors = libxml_get_errors();
+      $errorMessage = '';
+      foreach ($xmlErrors as $error) {
+        $errorMessage .= $error->message . '[File]: ' . $error->file .
```

Comment:
Using sprinf to assemble the error message might be cleaner, but otherwise looks solid.

Reply:
Done!